### PR TITLE
Chidera enhancement peer session flow

### PR DESCRIPTION
**Pull request by Zer0SZN**
## Preview
preview

### CODE REVIEW

File: development/src/composables/firebaseConfig/config.js
Comment (annonymousauthority) [inline development/src/composables/firebaseConfig/config.js:1]: Please remove this part

Comment (annonymousauthority) [reply development/src/composables/firebaseConfig/config.js:1]: This file shouldn't be part of the PR. I don't think it's doing anything

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Just slight changes

**Review by Remi-dee — DISMISSED** [review]
LGTM

**Review by Remi-dee — APPROVED** [review]

**Pull request merged**